config/kubernetes/sig-release/teams.yaml

```diff
@@ -74,92 +76,84 @@ teams:
     privacy: closed
   kubernetes-release-managers:
     description: People actively working on next k8s minor release, or who are patch
-      managers for any still-supported minor release.   Gives admin access to repos
+      managers for any still-supported minor release. Gives admin access to repos
       where branches must be created, etc., and write access to ones where label/PR
-      management is needed.  Remove users who are not actively doing this job.
+      management is needed. Remove users who are not actively doing this job.
     maintainers:
-    - idvoretskyi
-    - justaugustus
     - calebamiles
-    - jdumars
-    - tpepper
-    - AishSundar
+    - justaugustus
+    - tpepper # 1.13 PRM
     members:
-    - dougm
-    - jberkus
-    - idealhack
-    - jpbetz
-    - foxish
-    - krzyzacy
-    - hoegaarden
-    - mbohlool
-    - caesarxuchao
-    - aleksandra-malinowska
-    - enisoc
-    - MaciekPytel
-    - wojtek-t
+    - hoegaarden # 1.14 Branch Manager
+    - aleksandra-malinowska # 1.13 PRM
+    - feiskyer # 1.12 PRM
+    - foxish # 1.11 PRM
     - k8s-release-robot
     privacy: closed
   sig-release:
     description: SIG Release Members
     maintainers:
-    - spiffxp
```

Comment:
You may have to revert this. Because @spiffxp is an org owner, he's implicitly a maintainer of every team he's a member of. I think this will throw an error.

Reply:
Let me test this first..